packages/libs/src/sdk/oauth/OAuth.ts

```diff
@@ -353,7 +356,7 @@ export class OAuth {
     const decodedJwt = await this.verifyToken(event.data.token)
     if (decodedJwt?.data) {
       if (this.loginSuccessCallback) {
-        this.loginSuccessCallback(decodedJwt.data)
+        this.loginSuccessCallback(decodedJwt.data, event.data.token)
```

Comment:
@nicoback2 calling out this change for you since it modifies the public-facing API (in a backwards-compatible way). the alternative was to implement OAuth manually, which felt overkill because all i needed was to be able to send the token to the backend so it could verify instead of having to take the frontend's word for it

Reply:
works for me! thanks for calling this out.